godot-core/src/builtin/color_hsv.rs

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+use super::Color;
+
+/// HSV floating-number Color representation.
+///
+/// Godot's [`Color`] built-in type supports mainly RGBA floating point-based notation. `ColorHsv` supports manipulating its HSV
+/// representation by introducing conversion methods between itself and [`Color`].
+///
+/// `ColorHsv` *is not* a [`GodotType`](crate::builtin::meta::GodotType). To use it in properties expecting [`Color`], you need to convert
+/// it back to this type.
+///
+/// [`Color`] created by `ColorHsv::into::<Color>()` is equal to one created by `Color::from_hsv()`, but its execution time is approximately
```

Comment:
It's enough to make only the first occurrence of `Color` a link.

But you could maybe add a link to `ColorHsv` in the `Color` docs 🙂